test/test_helper.exs

```diff
@@ -17,7 +17,11 @@ four_hours = 240 * 60 * 1000
 ExUnit.configure(
   exclude: [disabled: true],
   module_load_timeout: four_hours,
-  timeout: four_hours)
+  timeout: four_hours,
+  formatters: case System.get_env("HONEYCOMB_FORMATTER") do
+    "true" -> [HoneycombFormatter, ExUnit.CLIFormatter]
+    _ -> [ExUnit.CLIFormatter]
+  end)
```

Comment:
Originally the honeycomb formatter was enabled without a code change using "mix test --formatter HoneycombFormatter --formatter ExUnit.CLIFormatter", but that doesn't work on Elixir 1.8's version of mix.